### DISCUSSION TITLE
For information. Compare quadriflow and quadwild-bimdf

### DISCUSSION THREAD

**Discussion by satabol** [cgg-bern/quadwild-bimdf#17] [General]
Hi. I build quadwild-bimdf and play with it. Here is a comparison [quadriflow](https://github.com/hjwdzh/QuadriFlow) (used by Blender) and quadwild-bimdf. Quadwild-bimdf has a better result:

![image](https://github.com/cgg-bern/quadwild-bimdf/assets/14288520/ece4eed4-2a9a-40f1-b4ee-8a8046f2c726)

Visualization for the comparizon - Blender-Sverchok.

Thank you!

**Comment by mheistermann** [comment]
Hi! Thanks for the feedback :-)

I see you work on Sverchok yourself -- are you considering adding quadwild-bimdf support to it?

**Reply by satabol** [reply]
Hi!
Yes I'm considering adding quadwild-bimdf to the Sverchok. Two things need to be done:
1. Convert settings/configs from files to a class to show settings in the UI of Sverchok Node. As example:
![image](https://github.com/cgg-bern/quadwild-bimdf/assets/14288520/d347aee6-1a48-438f-a750-73788dab423d)

2. Write a wrapper for Python and public it in the PyPi repo. We show additional packages in the Sverchok Settings:
![image](https://github.com/cgg-bern/quadwild-bimdf/assets/14288520/ff088b23-2901-4346-9e56-0f87cb5aea4e)
After package installed then node or some nodes can be used.

I can do both of them but I have to learn settings. Settings for quadwild-bimdf are not so simple like a quadriflow (as example). Do you know where to find a documentation for settings (1,2,3,4):
![image](https://github.com/cgg-bern/quadwild-bimdf/assets/14288520/4c31e391-e957-4964-91e7-6aa48c86fda9)

I can find parameters in the code:
![image](https://github.com/cgg-bern/quadwild-bimdf/assets/14288520/8bb47b4a-4cde-4e51-9003-d02c3debd715)
but I do not understand it. ))) A node in Sverchok must has a documentation for parameters. As example: [Voronoi on Mesh](https://nortikin.github.io/sverchok/docs/nodes/spatial/voronoi_on_mesh.html)
